LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceAPIService.cs

```diff
@@ -20,14 +21,17 @@ public sealed class LoRaDeviceAPIService : LoRaDeviceAPIServiceBase
     {
         private readonly IServiceFacadeHttpClientProvider serviceFacadeHttpClientProvider;
         private readonly ILogger<LoRaDeviceAPIService> logger;
+        private readonly Counter<int> deviceLoadRequests;
 
         public LoRaDeviceAPIService(NetworkServerConfiguration configuration,
                                     IServiceFacadeHttpClientProvider serviceFacadeHttpClientProvider,
-                                    ILogger<LoRaDeviceAPIService> logger)
+                                    ILogger<LoRaDeviceAPIService> logger,
+                                    Meter meter)
             : base(configuration)
         {
             this.serviceFacadeHttpClientProvider = serviceFacadeHttpClientProvider;
             this.logger = logger;
+            this.deviceLoadRequests = meter?.CreateCounter<int>(MetricRegistry.DeviceLoadRequests);
```

Comment:
[nit] same question here, maybe we can not use safe navigation, this would give us slightly more guarantees when executing the tests